tsconfig.json

```diff
@@ -1,13 +1,23 @@
 {
   "compilerOptions": {
     "target": "esnext",
-    "lib": ["esnext", "dom"],
+    "lib": [
+      "esnext",
+      "dom"
+    ],
     "moduleResolution": "node",
     "strict": true,
-    "declaration": false,
+    "declaration": true,
     "removeComments": true,
     "esModuleInterop": true,
+    "allowImportingTsExtensions": true,
+    "emitDeclarationOnly": true
   },
-  "include": ["./lib/**/*.ts"],
-  "exclude": ["**/*.spec.ts", "**/*.test.ts"],
+  "include": [
+    "./lib/**/*.ts"
+  ],
+  "exclude": [
+    "**/*.spec.ts",
+    "**/*.test.ts"
+  ],
```

Comment:
[note] Since this is an open source library, I would include tests as part of the type checking as well as an extra sanity check.

Reply:
I did not intentionally change this - it's only a formatting diff that you are seeing.

I agree. It's a super-valid suggestion.

Reply:
aa700a1